module.bcl2fastq.nf

```diff
@@ -0,0 +1,37 @@
+process bcl2fastq {
+    cache 'lenient'
```

Comment:
I would recommend removing the `cache` setting from each individual process definition. It's much better to set that in the config than it is to hardcode it in every process.